var/spack/repos/builtin/packages/bazel/package.py

```diff
@@ -17,6 +17,8 @@ class Bazel(Package):
     url      = "https://github.com/bazelbuild/bazel/releases/download/3.1.0/bazel-3.1.0-dist.zip"
 
     maintainers = ['adamjstewart']
+    version('4.0.0',  sha256='2b9999d06466815ab1f2eb9c6fc6fceb6061efc715b4086fa99eac041976fb4f')
```

Comment:
```suggestion
    version('4.0.0',  sha256='d350f80e70654932db252db380d2ec0144a00e86f8d9f2b4c799ffdb48e9cdd1')
```
`spack fetch bazel@4.0.0` gives a different checksum.